core/src/main/kotlin/ch/ergon/dope/resolvable/expression/unaliased/type/numberfunction/EulerExpression.kt

```diff
@@ -0,0 +1,14 @@
+package ch.ergon.dope.resolvable.expression.unaliased.type.numberfunction
+
+import ch.ergon.dope.DopeQuery
+import ch.ergon.dope.resolvable.expression.TypeExpression
+import ch.ergon.dope.validtype.NumberType
+
+class EulerExpression : TypeExpression<NumberType> {
+    override fun toDopeQuery() = DopeQuery(
+        queryString = "E()",
+        parameters = emptyMap(),
+    )
```

Comment:
I think this style might be more appropriate.

```kotlin
    override fun toDopeQuery() = DopeQuery(
        queryString = "E()",
        parameters = emptyMap(),
    )
```


```kotlin
    override fun toDopeQuery() = 
      DopeQuery(
        queryString = "E()",
        parameters = emptyMap(),
      )
```

Reply:
Nowhere else do we have this style. I will mark this as resolved.